app/config/ApplicationConfig.scala

```diff
@@ -51,8 +51,16 @@ trait ApplicationConfig {
 
 object ApplicationConfig extends ApplicationConfig with ServicesConfig {
 
+
+
   def getConf(key: String) = getConfString(key, throw new Exception(s"Could not find config '$key'"))
 
+  // PAYE tax year
+  lazy val payeTaxYear =  getConf("paye-tax-year")
```

Comment:
For PAYE they only keep one years worth of ATS data which is updated each year around November time. The current year now is 2019 (2018/19).  Next November or so it will change to 2020 (2019/20) at which time we will need to change the config.